### PR TITLE
Added a common misconceptions section to the Wiki

### PR DESCRIPTION
**Pull request by CuteCry**
Wrote up an article about some of the common issues people have with Windows in order to try and explain them. I like being detailed so this article might be longer than necessary, feel free to cut down on it if you'd like.

### CODE REVIEW

File: wiki/common-misconceptions.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Try loading up a memory intensive game, and taking a look at your total system memory utilization before and after launching the game. Let's say you are at 10 GB of total utilization before launching it, and the game is taking up about 6 GB. You'll see the total memory utilization only slightly creep up, possibly to 12 or 13 GB, not to 16 GB as you would expect. This is because Windows unloads stuff you don't need anymore to make room for the game's resources.
+
+RAM utilization is also dependent on how your RAM capacity - the more RAM you have, the more Windows uses to store frequently used code into standby memory. This is why some people with more memory notice higher utilization while some others with less memory notice significantly decreased utilization. Windows adapts based on how much memory you have and there is nothing to be concerned about if you notice high idle RAM utilization.
```
Comment (Erisa) [inline wiki/common-misconceptions.md:15]: The extra "how" here feels like a minor gramatical mistake/typo
```suggestion
RAM utilization is also dependent on your RAM capacity - the more RAM you have, the more Windows uses to store frequently used code into standby memory. This is why some people with more memory notice higher utilization while some others with less memory notice significantly decreased utilization. Windows adapts based on how much memory you have and there is nothing to be concerned about if you notice high idle RAM utilization.
```

**Review by Erisa — CHANGES_REQUESTED** [review]
Looks good to me, just one minor potential error.

It is indeed quite verbose but after reading through the entire page I realise all that verbosity is required to get across the points being delivered and the explanations and examples are all extremely well executed.

An extra thing that's needed is to add links to this page to [the wiki index](https://github.com/Microsoft-community/Community-website/blob/master/wiki/index.md) and to [the top mennu navigation](https://github.com/Microsoft-community/Community-website/blob/master/.vuepress/config.js#L53-L88) - happy to handle that myself, though you can also do it if you like.

File: wiki/common-misconceptions.md (hunk 1 above)
Comment (CuteCry) [reply wiki/common-misconceptions.md:15]: It is an error, thanks.

And oops I didn't realize I had to add those myself, doing it right now.

Comment (CuteCry) [reply wiki/common-misconceptions.md]: Everything should be in order now.

**Review by Erisa — APPROVED** [review]
All looks good, thank you for your contribution!

**Pull request merged**